kuma/users/checks.py

```diff
@@ -23,20 +26,44 @@ def stripe_check(app_configs, **kwargs):
         if not plan.active:
             errors.append(
                 Error(
-                    f"{settings.STRIPE_PLAN_ID} points to an inactive Strip plan",
+                    f"{settings.STRIPE_PLAN_ID} points to an inactive Stripe plan",
                     id=STRIPE_PLAN_INACTIVE_ERROR,
                 )
             )
     except stripe.error.StripeError as error:
         errors.append(
-            Error(f"unable to retrieve stripe plan: {error}", id=STRIPE_PLAN_ERROR)
+            Error(f"unable to retrieve Stripe plan: {error}", id=STRIPE_PLAN_ERROR)
         )
 
     try:
         create_missing_stripe_webhook()
     except stripe.error.StripeError as error:
         errors.append(
-            Error(f"unable get or create strip webhooks: {error}", id=STRIPE_PLAN_ERROR)
+            Error(
+                f"unable get or create Stripe webhooks: {error}", id=STRIPE_PLAN_ERROR
+            )
         )
 
     return errors
+
+
+def sendinblue_check(app_configs, **kwargs):
+    errors = []
+
+    if not settings.SENDINBLUE_API_KEY:
+        return errors
+
+    for id in [
+        settings.SENDINBLUE_PAYING_LIST_ID,
+        settings.SENDINBLUE_NOT_PAYING_LIST_ID,
+    ]:
+        response = sendinblue.request("GET", f"contacts/lists/{id}")
+        if not response.ok:
+            errors.append(
+                Error(
+                    f"Error when checking sendinblue list #{id}: {response.json()['message']}",
+                    id=SENDINBLUE_LIST_ERROR,
+                )
+            )
+
```

Comment:
I only typed in the API key and one of the list IDs. Silly me. This happened:
<img width="638" alt="Screen Shot 2020-05-07 at 8 30 03 AM" src="https://user-images.githubusercontent.com/26739/81294508-ffaa2680-903c-11ea-947d-cebe6f9eeb03.png">

Why is it complaining about `Listfoo1`? After all, I did create *one* of them:
<img width="1286" alt="Screen Shot 2020-05-07 at 8 31 04 AM" src="https://user-images.githubusercontent.com/26739/81294599-1fd9e580-903d-11ea-987d-bca0bcb6d8b7.png">